app/src/main/java/com/sdp13epfl2021/projmag/model/Tag.kt

```diff
@@ -11,7 +11,7 @@ data class Tag(var name: String) {
     private var accepted : Boolean = false
     init{
 
-        name = name.toLowerCase(Locale.ROOT).replace("\\s+".toRegex(), "")
+        name = name.replace("\\s+".toRegex(), "")
```

Comment:
It was already accepted in main, but is it really correct to remove space ?
For example the tag "Machine Learning" can't exist because it will be modified to "MachineLearning" with this replace.